pkgs/build-support/bintools-wrapper/setup-hook.sh

```diff
@@ -22,8 +22,10 @@ bintoolsWrapper_addLDVars () {
         # Python and Haskell packages often only have directories like $out/lib/ghc-8.4.3/ or
         # $out/lib/python3.6/, so having them in LDFLAGS just makes the linker search unnecessary
         # directories and bloats the size of the environment variable space.
-        local -a glob=( $1/lib/lib* )
-        if [ "${#glob[*]}" -gt 0 ]; then
+        local -a glob1=( $1/lib/*.so )
```

Comment:
Should we also detect `/lib/*.so.*` ?

Reply:
I think your right. I must have assumed that a symlink would exist. Which isn't the case on my fedora install.